lib/node_modules/@stdlib/string/next-grapheme-cluster-break/lib/main.js

```diff
@@ -63,15 +66,28 @@ function nextBreak( str, idx ) {
 	var next;
 	var len;
 	var mid;
+	var idx;
 	var i;
 
+	if ( !isString( str ) ) {
+		throw new TypeError( 'invalid argument. First argument must be a string. Value: `' + str + '`.' );
+	}
+	if ( arguments.length > 1 ) {
+		if ( !isInteger( fromIndex ) ) {
+			throw new TypeError( 'invalid argument. Second argument must be an integer. Value: `' + fromIndex + '`.' );
+		}
+		idx = fromIndex;
+	} else {
+		idx = 0;
+	}
 	len = str.length;
 	if ( idx < 0 ) {
-		return 0;
+		idx += len;
```

Comment:
```suggestion
		idx += len;
		if ( idx < 0 ) {
			idx = 0;
		}
```

I believe we need to bounds clip here. If the string length is `10` and `fromIndex` is `-20`, then after adding the string length the `idx` is still less than `0`.